subtrack.Tests/UtilitiesTests/GetDueClassTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using subtrack.MAUI.Utilities;
+
+namespace subtrack.Tests.UtilitiesTests;
+
+public class GetDueClassTests
```

Comment:
Might be a good idea to have this named CssUtil and have all tests in here since they are most likely going to be pure functions

Just a thought, i like the namespace Utilities since the file itself is going to be suffixed with "Tests", that way its similar to the actual app's structure

Reply:
Renamed class to CssUtilTests and changed the namespace to Utilities as suggested in latest commit.